.github/workflows/prepare-bundled-themes.yml

```diff
@@ -0,0 +1,157 @@
+      - 'src/wp-content/themes/twentynineteen/**'
+      - 'src/wp-content/themes/twentytwenty/**'
+      - 'src/wp-content/themes/twentytwentyone/**'
+      # Changes to this workflow file should always verify success.
+      - '.github/workflows/prepare-bundled-themes.yml'
+  pull_request:
+    branches:
+      - trunk
+      - '3.[89]'
+      - '[4-9].[0-9]'
+    paths:
+      # Changes to any themes with a build script should be confirmed.
+      - 'src/wp-content/themes/twentynineteen/**'
+      - 'src/wp-content/themes/twentytwenty/**'
+      - 'src/wp-content/themes/twentytwentyone/**'
+      # Changes to this workflow file should always verify success.
+      - '.github/workflows/prepare-bundled-themes.yml'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'The branch to create ZIP files from'
+        required: true
+        type: string
+        default: 'trunk'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name for pull requests
+  # or the commit hash for any other events.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  # Prepares bundled themes for release.
+  #
+  # Performs the following steps:
+  # - Checks out the repository.
+  # - Installs Node.js (only when theme has a build process)
+  # - Installs npm dependencies (only when theme has a build process)
+  # - Runs the theme build script (only when theme has a build process)
+  # - Ensures version-controlled files are not modified or deleted.
+  # - Creates a ZIP file of the theme for wordpress.org.
+  # - Uploads the ZIP file as a workflow artifact.
+  bundle-theme:
+    name: Prepare ${{ matrix.theme }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        theme: [
+            'twentytwentythree',
+            'twentytwentytwo',
+            'twentytwentyone',
+            'twentytwenty',
+            'twentynineteen',
+            'twentyseventeen',
+            'twentysixteen',
+            'twentyfifteen',
+            'twentyfourteen',
+            'twentythirteen',
+            'twentytwelve',
+            'twentyeleven',
+            'twentyten'
+        ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && inputs.branch || github.ref }}
+
+      - name: Install Node.js
+        uses: actions/setup-node@8c91899e586c5b171469028077307d293428b516 # v3.5.1
+        if: ${{ contains( fromJson( '["twentynineteen", "twentytwenty", "twentytwentyone"]' ), matrix.theme ) }}
```

Comment:
Is it possible to set an environment variable for the workflow equal to `${{ contains( fromJson( '["twentynineteen", "twentytwenty", "twentytwentyone"]' ), matrix.theme ) }}`?

There are four repetitions so it seems enough that one might be missed in an update.